services/src/Microsoft.Azure.IIoT.Services.Processor.Onboarding/src/Properties/launchSettings.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
Please remove

Reply:
done, no idea why was this created, by bad that I did not notice when adding it

Reply:
Sometimes this file is wanted/needed: https://stackoverflow.com/a/60383204/569998